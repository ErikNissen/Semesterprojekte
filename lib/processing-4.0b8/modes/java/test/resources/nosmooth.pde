version https://git-lfs.github.com/spec/v1
oid sha256:b52aee5c239233fda5d369ab2d8a84e218dd955b22d1759699c00a849526574e
size 175
