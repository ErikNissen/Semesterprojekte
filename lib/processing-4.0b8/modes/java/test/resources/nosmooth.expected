version https://git-lfs.github.com/spec/v1
oid sha256:61b81208c8b52707fe0bee439b08f07e8f4a12638edaa214daf5c3eb8130d27d
size 958
